packages/vendure-plugin-invoices/src/ui/invoices.component.ts

```diff
@@ -148,6 +98,7 @@ export class InvoicesComponent implements OnInit {
     this.form = this.formBuilder.group({
       enabled: ['enabled'],
       templateString: ['templateString'],
+      orderCode: ['99SWXHRE9D58LJLC'],
```

Comment:
Should this be in here?

Reply:
yeah it shouldn't be there, will remove it